sdk/metrics/build.gradle.kts

```diff
@@ -14,12 +16,14 @@ otelJava.moduleName.set("io.opentelemetry.sdk.metrics")
 
 dependencies {
     api(project(":api:metrics"))
+    api(project(":context"))
```

Comment:
do we need context at this point in the work?

Reply:
No, think this is an artifact of how I fragmented the branch. Removed.